backend/app/api/api_v1/endpoints/onboarding.py

```
@@ -105,11 +106,16 @@ async def get_onboarding_templates(
 @router.post("/preview", response_model=OnboardingPlanResponse)
 async def preview_onboarding_plan(
     payload: OnboardingPlanRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.16.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)